src/player/player.c

```diff
@@ -141,10 +141,17 @@ void playerRender(void* data, struct DynamicRenderDataList* renderList, struct R
     );
 }
 
+void playerHandleSweptCollision(struct CollisionObject* object, float velocityDot) {
+    // struct Player* player = object->data;
```

Comment:
Nitpick: commented out code can be removed (looks like it's from before you updated the function signature).